roles/openjdk/tasks/main.yml

```diff
@@ -1,27 +1,22 @@
 ---
 # tasks file for roles/openjdk
-- name: openjdk | install java (package)
+- name: Openjdk | install java (package)
   ansible.builtin.apt:
     name: "{{ java_openjdk_package }}"
     state: present
-  when:
-    - running_on_server
+    update_cache: true
```

Comment:
Didn't we remove this to prevent failures related to <random_package>? I'm okay with reinstating it if we need it, but I'm having a minor flashback.